models/models.js

```diff
@@ -47,9 +48,46 @@ const patchReviewVotes = (id, votes) => {
     });
 };
 
+const fetchReviews = (categoryNeeded) => {
+  const basicQuery = `SELECT reviews.review_id, title, designer, review_img_url, reviews.votes, category, owner, reviews.created_at, COUNT(comments.comment_id) AS comment_count
+  FROM reviews 
+  LEFT JOIN comments 
+  ON reviews.review_id = comments.review_id`;
+
+  const categoryQuery = ` WHERE category = $1`;
+
+  const groupQuery = ` GROUP BY reviews.review_id
+  ORDER BY reviews.created_at DESC;`;
+
+  if (categoryNeeded === undefined) {
+    return db.query(basicQuery + groupQuery).then(({ rows }) => {
+      return rows;
+    });
+  }
+
+  let categories = [];
```

Comment:
see [nc notes for optional where clauses](https://notes.northcoders.com/courses/js-back-end/complex-queries) to achieve this ticket more efficiently

Think your future-proofing the categories is pretty cool though, even if, for now, you can achieve the category check with an array of improved categories.includes()